pyramda/iterable/reject.py

```diff
@@ -0,0 +1,14 @@
+from pyramda.function.curry import curry
+from . import filter
+
+
+@curry
+def reject(p, xs):
+    """
+    Acts as a complement  of `filter`
+
+    :param p: predicate
+    :param xs: Iterable. A sequence, a container which supports iteration or an iterator
+    :return: list
+    """
+    return list(set(xs) - set(filter(p, xs)))
```

Comment:
Good catch! My current implementation is not correct.
There are two ways I think we can do this -  in term of the filter method as you mentioned:
` return list( x for x in xs if x not in filter(p, xs))`

or use the builtin itertools module
`
return list(itertools.ifilterfalse(p, xs))
`
What are your thoughts on the second approach? One downside of the second approach is that itertools.ifilterfalse is not curryable by default  so it does not directly fit in with  general theme of  ramda I think.